cdc/http_handler.go

```diff
@@ -285,3 +287,33 @@ func handleAdminLogLevel(w http.ResponseWriter, r *http.Request) {
 
 	writeData(w, struct{}{})
 }
+
+// Use to change gcBLT
+func (s *Server) handleConfigGCTimeToBlock(w http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
+		writeError(w, http.StatusBadRequest, cerror.ErrSupportPostOnly.GenWithStackByArgs())
+		return
+	}
+	s.ownerLock.Lock()
+	defer s.ownerLock.Unlock()
+	if s.owner == nil {
+		handleOwnerResp(w, concurrency.ErrElectionNotLeader)
+		return
+	}
+
+	err := req.ParseForm()
+	if err != nil {
+		writeInternalServerError(w, cerror.WrapError(cerror.ErrInternalServerError, err))
+		return
+	}
+
+	gcBLTStr := req.Form.Get(APIOpVarGcBLT)
```

Comment:
In design, the input of `/server/config` should be a JSON, but not an HTTPRequestForm

Reply:
ok.

Reply:
Will this be an official public API?